tests/Filesystem/FilesystemTest.php

```diff
@@ -383,7 +383,7 @@ public function testSharedGet()
             $result *= $status;
         }
 
-        $this->assertTrue($result === 1);
+        $this->assertEquals(1, $result);
```

Comment:
This should be assertSame (===)
As assertEquals does not take types into cosideration